patches/tModLoader/Terraria/NPC.cs.patch

```diff
@@ -2261,9 +2340,18 @@
  
  		UpdateNPC_BuffFlagsReset();
 +		NPCLoader.ResetEffects(this);
++		UpdateNPC_UpdateGravity();
  		UpdateNPC_BuffSetFlags();
  		UpdateNPC_SoulDrainDebuff();
  		UpdateNPC_BuffClearExpiredBuffs();
+ 		UpdateNPC_BuffApplyDOTs();
+ 		UpdateNPC_BuffApplyVFX();
+ 		UpdateNPC_BloodMoonTransformations();
+-		UpdateNPC_UpdateGravity(out var maxFallSpeed);
++		//UpdateNPC_UpdateGravity(out var maxFallSpeed) // TML: #AccessibleNPCGravity, removed out parameter
```

Comment:
Use block comment around the line, and note that it was moved to run with `NPCLoader.ResetEffects`